test/test_torchscript_consistency.py

```diff
@@ -209,7 +197,7 @@ def func(tensor):
 
     def test_lfilter(self):
         if self.dtype == torch.float64:
-            pytest.xfail("This test is known to fail for float64")
+            raise unittest.SkipTest("This test is known to fail for float64")
```

Comment:
I'd make this a decorator then and have separate tests for each dtype

Reply:
I think that's outside of the scope of this PR.